src/components/Retip.vue

```diff
@@ -250,20 +237,11 @@ export default {
     padding: 1rem;
     position: absolute;
 
-    .form-control {
-      color: $custom_links_color;
+    div {
+      display: flex;
 
-      &:focus {
-        box-shadow: none;
-      }
-
-      &[type=number]:focus {
-        border-right: none;
-      }
-
-      &[type=number]:focus ~ .input-group-append .input-group-text {
-        border: .05rem solid $custom_links_color;
-        border-left: none;
+      .input-group /deep/ input.form-control {
```

Comment:
`/deep/` is some experimental selector that is deprecated, is this some boostrap junk you need to override?

Reply:
Actually, Vue has it's own `deep` that doesn't look like deprecated
https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors

Reply:
As I know `deep` selectors are deprecated in native web components, in this case, `deep` shouldn't leak to client-side.

Reply:
Good to know. However I was actually asking isn't it an overshoot to use so complex selector for such a simple case?